_tools/release/02_create_pr.ts

```diff
@@ -35,7 +35,7 @@ function getPrBody() {
     `Please ensure:\n` +
     `- [ ] Version in version.ts is updated correctly\n` +
     `- [ ] Releases.md is updated correctly\n` +
-    `- [ ] All the tests in this branch have been run against the CLI release being done (\`../deno/target/release/deno test -A --unstable\`)\n\n` +
+    `- [ ] All the tests in this branch have been run against the CLI release being done (\`../deno/target/release/deno task test\`)\n\n` +
```

Comment:
Should we also get people running `deno task node:test`?

Reply:
I think so. `deno test -A --unstable` equivalent is `deno task test` + `deno task node:unit` + `deno task node:test`.